src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -52,12 +52,13 @@ import { DatagridColumnChanges } from './enums/column-changes.enum';
       </ul>
       <div class="switch-footer">
         <ng-content select="clr-dg-column-toggle-button"></ng-content>
-        <clr-dg-column-toggle-button *ngIf="!customToggleButton">Select All</clr-dg-column-toggle-button>
+        <clr-dg-column-toggle-button *ngIf="!customToggleButton">{{commonStrings.selectAll}}</clr-dg-column-toggle-button>
```

Comment:
Same here? We no longer need to use `*ngIf` and `@ContentChild`

Reply:
This is a non-breaking change for the deprecation, which gives the consumers a cycle to migrate to common strings patterns. It should remain unless you are voting to make the breaking change now?

Reply:
I vote for non-breaking. I'm not 💯 this is even a valid use case but, I'm still wondering if a consumer decides/needs to override the common strings just for one datagrid.

Reply:
This is not something we support anywhere with the string service, if you reuse any component with the string its always the same. If this use case comes up, we'd have to design a different solution, but for now I say its not valid.